_episodes/09-SWAN.md

```diff
@@ -0,0 +1,116 @@
+---
+title: "Using data in CloudStor"
+teaching: 5
+exercises: 5
+questions:
+- "How do you work in Jupyter Notebooks with data stored in CloudStor?"
+objectives:
+- "Use data stored in Cloudstor in Jupyter Notebooks"
+activity:
+- "Bring a dataset into a notebook"
+keypoints:
+ - "This is where the fun begins!"
+---
+
+# Using data in CloudStor
+
+CloudStor's SWAN (Service for Web-based Analysis) presents the [Jupyter Lab](https://towardsdatascience.com/jupyter-lab-evolution-of-the-jupyter-notebook-5297cacde6b) environment.
+
+Once the SWAN session has started, a file navigator is shown on the left and a work area to the right. The work area allows you to open many notebook files and terminals in tabs and/or side by side to your liking.
+
+When you open SWAN you are taken to the default location for your notebooks which is called a 'scratch' folder.
+
+The file system in SWAN is a little different to your CloudStor folder. We recommend you use  the SWAN home folder (**/scratch**) as the default space to work in rather than your CloudStor  folder (**/scratch/cloudstor**). In SWAN you will see a sub-folder called “cloudstor” which connects to your CloudStor data.
```

Comment:
```suggestion
The file system in SWAN is a little different to your CloudStor folder. We recommend you use  the SWAN home folder (**/scratch**) as the default space to work in rather than your CloudStor  folder (**/scratch/cloudstor**). In SWAN you will see a sub-folder called "cloudstor" which connects to your CloudStor data.
```
The quotes are a bit weird...

Reply:
Perhaps I can use the notations for code there instead - the way you suggested presenting /scratch.